ember.md

```diff
@@ -98,6 +100,40 @@ get(this, 'isDestructured'); //true
 set(someObject, 'isUpdated', true);
 get(someObject, 'isUpdated'); //true
 ```
+
+<a name="data"></a>
+## Data
+
+<a name="data--data-binding"></a>
+### 3.1 Data Binding
+
+Any new code written should only contain one-way data-binding
+
+> Why? One of our Front End principles is DDAU, data down actions up. Also, two-way data-binding can have unexpected side effects.
```

Comment:
I think the principles part can be removed. But it would be good to expand on what those unexpected effects can be